pkg/portal/cluster.go

```diff
@@ -190,3 +195,56 @@ func (p *portal) machineSets(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	_, _ = w.Write(b)
 }
+
+func (p *portal) statistics(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	duration := r.URL.Query().Get("duration")
+	endTimeString := r.URL.Query().Get("endtime")
+	endTime, err := time.Parse(time.RFC3339, endTimeString)
+	if err != nil {
+		p.internalServerError(w, err)
+		return
+	}
+	apiVars := mux.Vars(r)
+	subscription := apiVars["subscription"]
+	resourceGroup := apiVars["resourceGroup"]
+	clusterName := apiVars["clusterName"]
+	statisticsType := apiVars["statisticsType"]
+	resourceID := strings.ToLower(
+		fmt.Sprintf(
+			"/subscriptions/%s/resourceGroups/%s/providers/Microsoft.RedHatOpenShift/openShiftClusters/%s",
+			subscription, resourceGroup, clusterName))
+
+	promQuery, err := cluster.GetPromQuery(statisticsType)
+	if err != nil {
+		p.internalServerError(w, err)
+		return
+	}
+	prom := prometheus.New(p.log, p.dbOpenShiftClusters, p.dialer) //, p.authenticatedRouter)
```

Comment:
isn't this comment a leftover of some kind?

Reply:
yes, my bad, removed it and pushed the changes.